app/helpers/date_helper.rb

```diff
@@ -23,4 +23,13 @@ def date_in_user_time_zone(datetime)
     end
   end
 
+  # show date in the time zone specified
+  # note: this does *not* append timezone and does *not* reflect user preferences
+  def date_in_zone(time, zone = nil)
+    zone ||= Time.zone.name
+    return I18n.t("time.no_date_specified") if time.blank?
```

Comment:
It might not be easy for Translation to handle cases where `no_date_specified` is filled in where a sentence expects an actual date. I think we can simply return nil if time is not provided. It's up to the views to deal with nil cases correctly.

(For drafts, we use `created_at` which should always be set, so we should be fine.)

Reply:
Just to make sure, would it be fine if it would be a bit different from `time_in_zone`? `time_in_zone` has a similar string return value for "no time specified": https://github.com/otwcode/otwarchive/blob/4d661d5cf70afb32dfa6df84419b3de6b9dd6e27/app/helpers/application_helper.rb#L307

I have no personal preference, I just tried to make `date_in_zone` similar to `time_in_zone`.

Reply:
Yeah, it's fine if they're different. We'll leave `time_in_zone` as is but make `date_in_zone` simpler.